### DISCUSSION TITLE
First....

### DISCUSSION THREAD

**Discussion by RJ-Make** [OpenSageTV/Community#3] [General]
JK, Testing Looks good to me. Thanks!

**Comment by jusjoken** [comment]
Excellent.  I believe you are not a member of the OpenSageTV organization so looks like any member of GitHub can post with the current settings

**Reply by RJ-Make** [reply]
> Excellent.  I believe you are not a member of the OpenSageTV organization so looks like any member of GitHub can post with the current settings 

Only a member of the SageTV forums.

**Reply by RJ-Make** [reply]
Looks like searches work pretty good, albeit little information to test on. 🤪

**Reply by jusjoken** [reply]
With GitHub names different from sage forum names its going to be difficult to know who's who..

**Reply by jwittkoski** [reply]
Test Post

**Reply by jwittkoski** [reply]
Re: GitHub vs. SageTV Forum names - maybe we ask that people manually add a "sig" at the end of their posts with their SageTV forum name for a while? I can't see any other easy way to handle that.

**Reply by wnjj** [reply]
I’m wnjj on both platforms so I guess I’ve got it easy. :)

**Reply by nyplayer** [reply]
Looks good to me ...

**Reply by Narflex** [reply]
I'm always Narflex. :D

**Reply by Rob23-sagetv** [reply]
Setup looks good.  I tested creating a new GitHub account and didn't have an issue.  My username on the SageTV forum was Rob23.  Thank you for setting this up jusjoken!

**Reply by FoggySageTV** [reply]
(was "Foggy")
I'm using my account I set up in 2013 to get access to the MiniClient.
